javascript/chronometer.js

```diff
@@ -1,39 +1,53 @@
 // Constructor
-// function Chronometer() {
 
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+}
+
+Chronometer.prototype.startClick = function () {
+    intervalId = setInterval(function(){
+        chronometer.currentTime += 1;
+        chronometer.setTime();
+     }, 1000);
+};
```

Comment:
You're using "chronometer" instance of the Chronometer constructor as an object reference for currentTime and setTime. It works but it's not ideal because you have to consider the cases where there are several instances of the Chronometer (let's say chronometer1, chronometer2, etc.). 
A better practice would be to use arrow functions (where this refers to the object outside of the function).